scripts/base/frameworks/intel/main.zeek

```diff
@@ -497,6 +513,9 @@ function insert_meta_data(item: Item): bool
 # indicates whether the item might be new for other nodes.
 function _insert(item: Item, first_dispatch: bool &default = T)
 	{
+	if ( ! hook insert_item(item) )
```

Comment:
Placing this here seems almost equivalent with the existing `filter_item()` hook.

It also doesn't work as you require. E.g. if one calls `Intel::insert()` with the same item twice, this hook is invoked twice, even though there's only one entry in the data_store / min_data_store.